bundles/Planning/Resources/translations/messages.da.yaml

```diff
@@ -0,0 +1,19 @@
+planning:
+    board:
+        headline: Boards
+        filter_placeholder: Filtrér
+        loading: Indlæser...
+        title: 'Boards - Planning - ITK Jira'
+    choose_another_board: '<- Vælg andet board'
```

Comment:
https://www.fileformat.info/info/unicode/char/2190/index.htm
Are we sure that this will always be shown to the right of “choose board”?